google-map-marker.html

```diff
@@ -192,7 +195,8 @@
       longitude: {
         type: Number,
         value: null,
-        reflectToAttribute: true
+        reflectToAttribute: true,
```

Comment:
what's the use case for these to reflect? You only need to reflect when you want to style something with an attribute selector: `google-map-marker[longitude="123"]`